corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -113,13 +101,17 @@ def handle(self, dry_run=False, **options):
             self.sql_class().__name__,
         ))
         for doc in get_all_docs_with_doc_types(self.couch_db(), [self.couch_doc_type()]):
-            logger.info("{}Looking at doc with key {}".format(log_prefix, self.doc_key(doc)))
+            logger.info("{}Looking at {} doc with id {}".format(
+                log_prefix,
+                self.couch_doc_type(),
+                doc["_id"]
+            )
```

Comment:
Missing close bracket.
```suggestion
            ))
```